draft-ietf-dnssd-srp-replication.xml

```diff
@@ -231,9 +231,17 @@
       <section anchor="startup">
         <name>Startup</name>
 
-        <t>When a partner starts SRP Replication, it enters the "startup" state and picks a random discovery time interval uniformly selected from the range [MIN_PARNTER_DISCOVERY_INTERVAL, MAX_PARTNER_DISCOVERY_INTERVAL]. The recommended minimum and maximum values are 4 and 7.5 seconds, respectively.</t>
+        <t>When a partner starts SRP Replication, it enters the "startup" state and picks a random discovery time interval uniformly selected from the range [MIN_PARNTER_DISCOVERY_INTERVAL, MAX_PARTNER_DISCOVERY_INTERVAL]. The recommended minimum and maximum values are 4 and 7.5 seconds, respectively. The new partner uses the discovery time to (1) discover and determine the preferred dataset ID, (2) find all existing SRPL partners, establish SRPL sessions with them and synchronize with them. The randomness introduced in the start time helps to handle the situation where a group of SRPL partners start around the same time, for example, triggered by a power outage.</t>
 
-        <t>When the discovery time interval expires, the SRPL partner transitions to the "routine operation" state (<xref target="routine-operation"/>) after it has either successfully synchronized with or NUM_DISCOVERY_SYNC_ATTEMPTS have occurred with each discovered partner. If the partner did not discover any other SRPL partners to synchronize with, it immediately transitions to the "routine operation" state.</t>
+        <t>When the discovery time interval expires, if the partner did not discover any other SRPL partners to synchronize with, it transitions to the "routine operation" state (<xref target="routine-operation"/>). If it discovers other SRPL partners, it can enter "routine operation" state after one of the following conditions are met for every partner it discovered during the discovery interval:</t>
+        <ul>
+          <li>The new partner has established an SRPL session with the discovered partner and successfully finished initial synchronization (<xref target="initial-synchronization"/>).</li>
```

Comment:
```suggestion
          <li>The new partner has established an SRPL session with a discovered partner and successfully finished initial synchronization (<xref target="initial-synchronization"/>).</li>
```

Reply:
I think we should keep "the" in this sentence since this rule applies to each discovered partner. We have this sentence right before this:
>  ... it can enter "routine operation" state after one of the following conditions are met for every partner it discovered during the discovery interval:
> 1) The new partner has established an SRPL session with the discovered partner and successfully finished initial synchronization...

Reply:
Ah, yes, I missed that.

Reply:
The right flow is to discuss the change on the mailing list, and then implement it here. The above discussion would normally be done on the mailing list. Otherwise it's an isolated conversation and you're not really getting the benefit of working group review.

Reply:
SGTM.

@abtink , let's propose all changes on the mailing list first.

Reply:
Please see https://github.com/dnssd-wg/draft-ietf-dnssd-srp-replication/pull/10#issuecomment-1535478461 below. I will post link t the PR at the end of same mailing chain.